basic/README.md

```diff
@@ -18,3 +18,251 @@ function.
     open http://localhost:3000/
 
 Runs the app from http://localhost:3000
+
+For more information see [Use Mapbox GL JS in a React app](https://docs.mapbox.com/help/tutorials/use-mapbox-gl-js-with-react/) 
+
+
+Documentation
+---
+
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
```

Comment:
If we're stripping out most of the bootstrapping from create react app, Want to drop this line?